pkg/scheduler/objects/nodesorting.go

```diff
@@ -31,6 +31,7 @@ import (
 type NodeSortingPolicy interface {
 	PolicyType() policies.SortingPolicy
 	SortNodes(nodes []*Node)
```

Comment:
is `SortNodes()` still needed?
My understanding is that now we just need to score the nodes

Reply:
I'm a little hesitant to remove this just yet... there may still be a need for this moving forward?